lib/vsc/install/ci.py

```diff
@@ -0,0 +1,185 @@
+#
+# Copyright 2019-2019 Ghent University
+#
+# This file is part of vsc-install,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/hpcugent/vsc-install
+#
+# vsc-install is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Library General Public License as
+# published by the Free Software Foundation, either version 2 of
+# the License, or (at your option) any later version.
+#
+# vsc-install is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Library General Public License for more details.
+#
+# You should have received a copy of the GNU Library General Public License
+# along with vsc-install. If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Generate configuration files for running CI tests.
+
+Run with: python -m vsc.install.ci
+
+@author: Kenneth Hoste (Ghent University)
+"""
+from __future__ import print_function
+from optparse import OptionParser
+import logging
+import os
+import sys
+
+
+JENKINSFILE = 'Jenkinsfile'
+JENKINSFILE_REVISION = 'Jenkinsfile@20191122-02'
+TOX_INI = 'tox.ini'
+TOX_INI_REVISION = 'tox.ini@20191122-01'
+
+logging.basicConfig(format="%(message)s", level=logging.INFO)
+LOG = logging.getLogger()
+
+
+def parse_options(args):
+    """Parse options."""
+    parser = OptionParser()
+
+    parser.add_option('-f', '--force', dest='force', action='store_true', help="Use force to overwrite existing files")
```

Comment:
why is this optional? you are always supposed to use it....

Reply:
Because I really don't like blindly overwriting files by default...
Usually the `Jenkinsfile` and `tox.ini` will be in a git repo, but not always (although I admit it's unlikely).
If you think this is pointless, then I can drop this option and just blindly overwrite.

In practice you'll always run `python -m vsc.install.ci -f`, indeed.

Reply:
i think this is pointless because it is meant for git repos, so any accidental overwrites are still not that bad

Reply:
dropped the `--force`, blindly overwriting now...